scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -1,45 +1,31 @@
 ---
 title: "SARS-CoV-2 variant report"
 date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
-output:
-  html_document:
-    theme: lumen
-    highlight: monochrome
-    code_folding: hide
-    depth: '3'
-    toc: yes
-    toc_float: yes
-    number_sections: yes
 params:
-  sample_name: ''
-  sigmut_db: ''
-  variants_dir: ''
-  vep_file: ''
-  snv_file: ''
-  sample_sheet: ''
+    @PARAMS_SAMPLE_NAME@
 
 ---
 
-```{r libs, include=FALSE, message = FALSE, warning = FALSE}
-knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
-library(knitr)
-library(dplyr)
-library(ggplot2)
-library(DT)
-library(tidyr)
-library(qpcR)
-library(stringr)
-library(magrittr)
+Read config.yml for the locations of sample sheet, pipeline output and sigmut loc and variants_dir
+```{r, child="_settings.Rmd"}
+```
+
+```{r, libs}
 library(base64url)
+library(magrittr)
 ```
 
 # Input Settings
 ```{r printInputSettings, echo = FALSE}
 sample_name <- params$sample_name
-sample_sheet <- data.table::fread(params$sample_sheet)
-location_sigmuts <- params$sigmut_db
+location_sigmuts <- sigmut_db
+
+vepfile <- file.path(variants_dir, paste0(sample_name, "_vep_sarscov2_parsed.txt"))
+snvfile <- file.path(variants_dir, paste0(sample_name, "_snv.csv"))
+sampledir <- variants_dir
```

Comment:
This is all old.  Please revert.  @mirifax and I worked on getting rid of most of these lines in our efforts to use `rmarkdown::render` instead of `render_site`.